src/main/resources/hudson/plugins/junitattachments/AttachmentTestAction/sidepanel.jelly

```diff
@@ -1,6 +1,7 @@
+<?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define"
 	xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
 	
-	<st:include from="${it.testObject.owner}" it="${it.testObject.owner}" page="sidepanel.jelly" />
+	<st:include from="${it.testObject.run}" it="${it.testObject.run}" page="sidepanel.jelly" />
```

Comment:
BTW I believe [`from` may be omitted when identical to `it`](https://github.com/stapler/stapler/blob/057819f66c990ee4d156fb52765d9e3414f62a4b/jelly/src/main/java/org/kohsuke/stapler/jelly/IncludeTag.java#L131-L162).